apps/frontend/src/hooks/aave/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const BIG_NUMBER_PRECISION_TWENTY_SEVEN = 27;
+export const BIG_NUMBER_PRECISION_EIGHTEEN = 18;
```

Comment:
This is not particular to aave, let's move it to `math.ts` and maybe rename to `RAY` and `WEI`